### ISSUE TITLE
The proposed fix for CA2208 is not consistent in the doc

### ISSUE THREAD

**Issue by longyuan1996** [dotnet/docs#44030]
### Type of issue

Other (describe below)

### Description

[Enter feedback here]

Hi, I'm trying to fix the CA2208 error in my project. But in the official documentation, I found that the fixes are totally different in the tip and the example.

In the `How to fix violations` tip, the parameter name is placed to the 2nd paramenter:
![Image](https://github.com/user-attachments/assets/902d24ef-e3fc-419e-90e5-266333b2dc0f)

But in the `Example` Session, the name is placed to the 1st parameter:
![Image](https://github.com/user-attachments/assets/b8acb6f0-5733-4898-86cf-36a53224ccda)

From the [ArgumentException Constructor (System) | Microsoft Learn](https://learn.microsoft.com/en-us/dotnet/api/system.argumentexception.-ctor?view=net-8.0#system-argumentexception-ctor(system-string-system-string)) the parameter should be in the 2nd parameter.

### Page URL

https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2208#how-to-fix-violations

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/fundamentals/code-analysis/quality-rules/ca2208.md

### Document Version Independent Id

cc2b5015-89a9-02ff-a585-fca3449dfa9a

### Article author

@gewarren

### Metadata

* ID: ff82be90-2d17-5880-4edd-a89b18a3ae2b 
* Service: **dotnet-fundamentals**

**Comment by BartoszKlonowski** [comment]
@gewarren Let me take a look into that and work on it if needed.

**Comment by BartoszKlonowski** [comment]
@gewarren Yes, I confirm this is the tooltip that requires an update of the screenshots.
The only question is: Is it OK to use the "Blue" theme for new ones (as the Visual Studio changed it's look after those years)?

**Comment by BartoszKlonowski** [comment]
One more thing: I don't think we will need the second screenshot any longer, as I can see that having single parameter of the type name matches one of the `ArgumentNullException` overloads:

```csharp
        public ArgumentNullException(string? paramName)
            : base(SR.ArgumentNull_Generic, paramName)
        {
            HResult = HResults.E_POINTER;
        }
```

**Comment by gewarren** [comment]
I believe the doc is actually correct as written. There's a subtle difference here because the parameter order is different between ArgumentException and ArgumentNullException, as noted in the article. Perhaps that part needs to be more prominent? Or maybe we just go ahead and switch the screenshot to the ANE exception.

**Comment by gewarren** [comment]
> One more thing: I don't think we will need the second screenshot any longer, as I can see that having single parameter of the type name matches one of the `ArgumentNullException` overloads:

This part is also okay as-is. The code fix appears for `ArgumentException`, where the single string argument constructor takes a message string, not a paramName string.
